tests/t333-basis.c

```diff
@@ -0,0 +1,18 @@
+/// @file
+/// Test that Hale-Trefethen transform  works
+/// \test Test that Hale-Trefethen transform works
+#include <ceed.h>
+#include <math.h>
+
+int main(int argc, char **argv) {
+    Ceed ceed;
+    CeedInt Q = 16;
+    CeedScalar rho;
+    CeedScalar *g, *g_prime;
+
+    CeedHaleTrefethenStripMap();
```

Comment:
We talked about checking by spot-checking accuracy relative to figure 7.1a. How would you test that? Take 20-point Gauss quadrature and integrate the function, then transform the quadrature points and weights, and integrate that way in $[-1, 1]$. The second should be more accurate.

![image](https://user-images.githubusercontent.com/3303/173156684-be65aea5-497a-4d3e-a66a-28046d5b80d8.png)